Classes/Plugins/RelatedListTool/RelatedListTool.php

```diff
@@ -154,12 +172,14 @@ public function getRelatedItems()
             $title = (string) $relatedItemXmlElement->xpath('mods:titleInfo/mods:title')[0];
             $docId = (string) $relatedItemXmlElement->xpath('mods:identifier[@type="' . $type . '"]')[0];
             $order = (string) $relatedItemXmlElement->xpath('mods:part/@order')[0];
+            $volume = (string) $relatedItemXmlElement->xpath('mods:part[@type="volume"]/mods:detail/mods:number')[0];
 
             $element = array();
             $element['type'] = $type;
             $element['title'] = $title;
             $element['docId'] = $docId;
-            $element['order'] = (is_numeric($order)) ? (int) $order : 0;
+            $element['order'] = (is_numeric($order)) ? (int) $order : PHP_INT_MAX;
+            $element['volume'] = (!empty($volume)) ? $volume : null;
 
             $relatedItems[$index] = $element;
         }
```

Comment:
Please note and fix this warning in current 1.3 release:
Core: Error handler (FE): PHP Warning: usort() expects parameter 2 to be a valid callback, class 'RelatedListTool' not found in /ext/dpf/Classes/Plugins/RelatedListTool/RelatedListTool.php line 167